cli/tsc/00_typescript.js

```diff
@@ -91955,7 +91955,7 @@ var ts;
                     // Set Accessors can have TypeScript modifiers and type annotations.
                     return visitSetAccessor(node);
                 case 256 /* SyntaxKind.FunctionDeclaration */:
-                    // Typescript function declarations can have modifiers, decorators, and type annotations.
+                    // TypeScript function declarations can have modifiers, decorators, and type annotations.
```

Comment:
This is what exists in the upstream TypeScript projects and is copied directly from their distributions.  If you want to fix this, you will need to raise a PR at https://github.com/microsoft/TypeScript.